sound/smooth_swing_v2.h

```diff
@@ -94,94 +111,123 @@ class SmoothSwingV2 : public SaberBasePassThrough {
         if (speed < smooth_swing_config.SwingStrengthThreshold) {
 #if 1
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
-            STDOUT.print("speed: ");
-            STDOUT.println(speed);
+           STDOUT.print("speed: ");
+           STDOUT.println(speed);
           }
 #endif
           break;
         }
         state_ = SwingState::ON;
         
       case SwingState::ON:
+        //check for AccentSwingThreshold, presence of accent swings and if the accent player is stopped (this prevents clipping)
+        if (speed >=smooth_swing_config.AccentSwingSpeedThreshold && accent_swings_present && !D.isPlaying() && (A.isPlaying() || B.isPlaying()))
```

Comment:
I don't think we need to check if A or B is playing since we're in SwingState::ON

Reply:
I thought so to, but I had accent swings playing while the saber was off until I added that check...

Reply:
I'm going to leave this check in for safety.  I don't think it's hurting anything and I don't get the accent swings when the saber is off anymore.